metadata/src/main/java/edu/unc/lib/dl/validation/MODSValidator.java

```diff
@@ -0,0 +1,122 @@
+/**
+ * Copyright 2008 The University of North Carolina at Chapel Hill
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package edu.unc.lib.dl.validation;
+
+import static edu.unc.lib.dl.xml.JDOMNamespaceUtil.SCHEMATRON_VALIDATION_REPORT_NS;
+
+import java.io.ByteArrayInputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.util.stream.Collectors;
+
+import javax.xml.transform.stream.StreamSource;
+import javax.xml.validation.Schema;
+
+import org.apache.commons.io.IOUtils;
+import org.jdom2.Document;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.xml.sax.SAXException;
+
+import edu.unc.lib.dl.schematron.SchematronValidator;
+
+/**
+ * Asserts that MODS complies with the XSD and controlled vocabularies,
+ * as per the configured schematron validator.
+ *
+ * @author bbpennel
+ * @author harring
+ *
+ */
+public class MODSValidator {
+    private static final Logger log = LoggerFactory.getLogger(MODSValidator.class);
+
+    private SchematronValidator schematronValidator = null;
```

Comment:
Nitpick, but these don't need to be initialized to null, they will be null by default.

Reply:
hey, i already nitpicked you on this! i left a comment asking why these were being set to null :-)